python/pyspark/pandas/namespace.py

```diff
@@ -285,6 +286,9 @@ def read_csv(
         Indicates the encoding to read file
     options : dict
         All other options passed directly into Spark's data source.
+    na_filter : bool
+        If na_filter is false missing values will remain as is otherwise it
+        will be converted to None. By default it will True
```

Comment:
Have done the changes . However , in case of Spark IMO , there wouldn't be any performance improvement , since it is using Univocity parsers which handle the null  when reading data.  Therefore I have not added performance related things in the documenataion